test/oli_web/live/user_actions_test.exs

```diff
@@ -0,0 +1,96 @@
+defmodule OliWeb.UserActionsTest do
+  use OliWeb.ConnCase
+  use Surface.LiveViewTest
+
+  import Oli.Factory
+
+  alias OliWeb.Users.Actions
+
+  describe "email confirmation" do
```

Comment:
I love testing the components isolated. Just a reminder that we have a test for the authors detail view (decide it by your own if makes sense to add tests there): https://github.com/wyeworks/oli-torus/blob/master/test/oli_web/live/admin_live_test.exs#L579